datasets/RES17_noord_holland_zuid/RES17_noord_holland_zuid.derived.ad

```diff
@@ -140,38 +139,23 @@
 - coast_line = 16.718
 - interconnector_capacity = 60000.0
 - total_land_area = 1222.73
-- number_of_buildings = 142127.0
+- number_of_buildings = 53512.0
 - number_of_cars = 852467.0
 - number_of_residences = 1004649.0
 - number_of_inhabitants = 2122663.0
 - offshore_suitable_for_wind = 6199.422321
 - residences_roof_surface_available_for_pv = 16.22878
 - buildings_roof_surface_available_for_pv = 25.284986
-- energetic_emissions_other_ghg_agriculture = 0.0
-- non_energetic_emissions_co2_agriculture_manure = 0.0
-- non_energetic_emissions_co2_agriculture_soil_cultivation = 0.0
-- non_energetic_emissions_other_ghg_agriculture_fermentation = 0.0
-- non_energetic_emissions_other_ghg_agriculture_manure = 0.0
-- non_energetic_emissions_other_ghg_agriculture_soil_cultivation = 0.0
-- non_energetic_emissions_other_ghg_agriculture_other = 0.416009
-- energetic_emissions_other_ghg_households = 0.0
-- energetic_emissions_other_ghg_buildings = 0.071296
-- energetic_emissions_other_ghg_transport = 0.182204
-- non_energetic_emissions_co2_chemical_industry = 0.0
-- non_energetic_emissions_co2_other_industry = 0.0
-- non_energetic_emissions_co2_waste_management = 0.0
-- non_energetic_emissions_other_ghg_chemical_industry = 0.0
-- non_energetic_emissions_other_ghg_other_industry = 0.561904
-- non_energetic_emissions_other_ghg_waste_management = 0.0
-- energetic_emissions_other_ghg_industry = 0.0
-- energetic_emissions_other_ghg_energy = 0.0
-- indirect_emissions_co2 = 0.0
+- other_emission_agriculture = 0.416009
```

Comment:
These attributes are outdated. Exported using an old version of the data manager / data manager database?

Reply:
Exported with updated dataset manager and database.